website_odoo_debranding/templates/disable_odoo.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
It seems to me that this line should be not removed. I'm checking the tech documentation and I don't see any indication for removing this.

Reply:
In general XML declarations in descriptions are deprecated since Odoo 17. I think this is related primarly to module descriptions. Shouldn´t this apply to views as well, to have a straight handling of the encoding?

Reply:
So it would be better to leave the disable_odoo.xml untouched IMO.

Reply:
ok i will return this.
The bot will only take care of the index, when the dotfiles are updated on each repo. so not yet...